src/components/assessments/AssessmentCard.tsx

```diff
@@ -1,73 +1,38 @@
-import {Box} from '@mui/material'
+import {Card, CardActions} from '@mui/material'
 import CardContent from '@mui/material/CardContent'
 import {styled} from '@mui/material/styles'
 import Typography from '@mui/material/Typography'
-import createStyles from '@mui/styles/createStyles'
-import makeStyles from '@mui/styles/makeStyles'
 import {FunctionComponent} from 'react'
-import {theme} from '../../style/theme'
+import {latoFont, theme} from '../../style/theme'
 import {Assessment} from '../../types/types'
 import AssessmentImage from './AssessmentImage'
 
-const CardContainer = styled(Box, {label: 'cardContainer'})(({theme}) => ({
-  backgroundColor: '#FFFFFF',
-
-  boxShadow: '0px 1px 4px rgba(0, 0, 0, 0.15)',
+const StyledCard = styled(Card)(({theme}) => ({
+  width: theme.spacing(28),
+  height: theme.spacing(47),
```

Comment:
I did something similar in [this PR](https://github.com/Sage-Bionetworks/synapse-web-monorepo/pull/267#discussion_r1218277137), but ultimately decided not to use the theme spacing for width/height, since I wasn't convinced that the component dimensions should change if the spacing was changed in the future, so used fixed values instead. Up to you whether that logic applies here!

Reply:
This is copy/pasted from the static site which was apparently more up-to-date? Really, it would be nice to have these use a shared source for the design of the assessment library but not today. ;)